### ISSUE TITLE
testing environment?

### ISSUE THREAD

**Issue by sweeneyb** [robodogs435frc/2015-Website-Redesign#19]
I was looking last night to review the progress, but I don't have a "free" box on which to install wordpress.  I looked up a virtualized box and found something that looks fairly simple: https://github.com/jalefkowit/vagrant-ansible-wordpress.  It was fairly easy to get the box up.

What do you guys think about redoing the readme.md file to be our own instructions about getting an environment up instead of the default wordpress page?  Might we want to have a db dump so that we can add that to the provisioning process?

Just thinking out loud to make it easier to get up and running.

**Comment by adamsong** [comment]
I am not sure how to make a db dump but I will work on the readme.

**Comment by sweeneyb** [comment]
Are you using mysql? Or mariaDb? Google for mysqldumb.  Other dbs should have a similar tool.

**Comment by adamsong** [comment]
MySQL

**Comment by sweeneyb** [comment]
Yeah.. wp uses prefixes for tables. Get all tables with that prefix and let us know the credentials you're using. That should be enough for everyone to get something running.https://dev.mysql.com/doc/refman/5.1/en/mysqldump.html

**Comment by adamsong** [comment]
DB Dump and readme are in pull request.
